src/components/common/symbol/crossSymbol.vue

```diff
@@ -0,0 +1,22 @@
+<script setup lang="ts">
+withDefaults(
+  defineProps<{
+    fillColor?: string
+    fontSize?: number
+    height?: number
+    width?: number
+  }>(),
+  {
+    fillColor: 'red',
+    fontSize: 42,
+    height: 18,
+    width: 18,
+  }
+)
+</script>
+
+<template>
+  <svg :height="height" :width="width">
+    <text x="0" :y="height + 1" :font-size="fontSize" :fill="fillColor">+</text>
```

Comment:
Seems a little less centered than the rest; Maybe `x="2" :y="height + 2"`?

Reply:
Good point thx